vmdb/app/controllers/catalog_controller.rb

```diff
@@ -1263,7 +1263,7 @@ def available_managers(template_id)
   end
 
   def available_templates
-    available_templates = OrchestrationTemplate.find(:all)
+    available_templates = OrchestrationTemplate.available
     @edit[:new][:available_templates] = []
     available_templates.each do |template|
       @edit[:new][:available_templates].push([template.name, template.id])
```

Comment:
You do not need the `available_templates` variable here.  This could move down to the each like and just be `OrchestrationTemplate.available.each do |template|`

I know you did not write this code, but it could be simplified to:
```ruby
def available_templates
  @edit[:new][:available_templates] = OrchestrationTemplate.available.collect {|template| [template.name, template.id]}.sort!
  @edit[:new][:template_id] = @record.orchestration_template.try(:id)
  @edit[:new][:manager_id] = @record.orchestration_manager.try(:id)
  available_managers(@record.orchestration_template.id) if @record.orchestration_template
end
```